user/languages/go.md

```diff
@@ -42,11 +42,11 @@ new to Travis CI please read our [Tutorial](/user/tutorial/) and
 
 You can use any tagged version of Go, a version with `x` in place of the minor
 or patch level to use the latest for a given major or minor version, or use
```

Comment:
This all seems quite convoluted (not just because of your update). What do you think about

```
You can use any tagged version of Go, a version with `x` in place of the minor
or patch level to use the latest for a given major or minor version, `stable` for the latest stable release, or `master` to get the latest version from source.

All go version management is handled by [gimme](https://github.com/travis-ci/gimme).

> When using `master` to get the latest development version from the [Go repository master branch](https://github.com/golang/go) the installation will take longer than the latest `stable` release.
```